src/azure-cli/azure/cli/command_modules/apim/custom.py

```diff
@@ -317,107 +344,83 @@ def import_apim_api(
-        resource.value = content_value
+        parameters.value = content_value
     elif specification_url is not None and specification_path is None:
-        resource.value = specification_url
+        parameters.value = specification_url
     elif specification_path is not None and specification_url is not None:
-        raise CLIError(
+        raise MutuallyExclusiveArgumentError(
             "Can't specify specification-url and specification-path at the same time.")
     else:
-        raise CLIError(
+        raise RequiredArgumentMissingError(
             "Please either specify specification-url or specification-path.")
 
     FORMAT_MAPPINGS = {
         ImportFormat.Wadl.value: {
             # specification_path is not none
-            True: ContentFormat.wadl_xml.value,
+            True: ContentFormat.WADL_XML.value,
             # specification_url is not none
-            False: ContentFormat.wadl_link_json.value
+            False: ContentFormat.WADL_LINK_JSON.value
         },
         ImportFormat.Swagger.value: {
-            True: ContentFormat.swagger_json.value,
-            False: ContentFormat.swagger_link_json.value
+            True: ContentFormat.SWAGGER_JSON.value,
+            False: ContentFormat.SWAGGER_LINK_JSON.value
         },
         ImportFormat.OpenApi.value: {
-            True: ContentFormat.openapi.value,
-            False: ContentFormat.openapi_link.value
+            True: ContentFormat.OPENAPI.value,
+            False: ContentFormat.OPENAPI_LINK.value
         },
         ImportFormat.OpenApiJson.value: {
-            True: ContentFormat.openapijson.value,
-            False: ContentFormat.openapi_link.value
+            True: ContentFormat.OPENAPI_JSON.value,
+            False: ContentFormat.OPENAPI_JSON_LINK.value
         },
         ImportFormat.Wsdl.value: {
-            True: ContentFormat.wsdl.value,
-            False: ContentFormat.wsdl_link.value
+            True: ContentFormat.WSDL.value,
+            False: ContentFormat.WSDL_LINK.value
         }
     }
 
     if specification_format in FORMAT_MAPPINGS:
-        resource.format = FORMAT_MAPPINGS[specification_format][specification_path is not None]
+        parameters.format = FORMAT_MAPPINGS[specification_format][specification_path is not None]
     else:
-        raise CLIError(
-            "Please provide valid value for specificationFormat: " + specification_format)
-
-    resource.protocols = protocols
-    resource.service_url = service_url
-    resource.display_name = display_name
-    resource.description = description
-    resource.subscription_required = subscription_required
-    resource.subscription_key_parameter_names = get_subscription_key_parameter_names(
-        subscription_key_header_name, subscription_key_query_param_name)
-    resource.api_version = api_version
-    resource.api_version_set_id = _get_vs_fullpath(api_version_set_id)
+        raise InvalidArgumentValueError(
```

Comment:
>Invalid [](http://example.com/codeflow?start=14&length=7)

Miss importing "InvalidArgumentValueError" which makes CI failed, please fix it.